### PR TITLE
sekindoUM for prebid1.0

### PR DESCRIPTION
**Pull request by nissSK**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

sekindoUM for prebid1.0

- test parameters for validating bids
```
 var adUnits = [{
      code: 'banner-ad-div',
      sizes: [[300, 250]],
      bids: [{
          bidder: 'sekindoUM',
          params: {
              spaceId: 14071
          }
      }]
    },
        {
      code: 'video-ad-div',
      sizes: [[640, 480]],
      bids: [{
          bidder: 'sekindoUM',
          params: {
              spaceId: 87812,
                          video:{
                                playerWidth:640,
                                playerHeight:480,
                                vid_vastType: 5 //optional
                          }
          }
      }]
    }
        ];

```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

### CODE REVIEW

File: modules/sekindoUMBidAdapter.js [hunk 1]
```diff
@@ -1,91 +1,113 @@
...
-var adloader = require('src/adloader.js');
-var adaptermanager = require('src/adaptermanager');
-
-function SekindoUMAdapter() {
-  function _callBids(params) {
-    var bids = params.bids;
-    var bidsCount = bids.length;
-
+import * as utils from 'src/utils';
+//  import {config} from 'src/config';
```
Comment (matthewlane) [inline modules/sekindoUMBidAdapter.js]: Line can be removed if not used

File: modules/sekindoUMBidAdapter.js [hunk 2]
```diff
@@ -1,91 +1,113 @@
...
+      };
+    });
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse, bidRequest) {
```
Comment (matthewlane) [inline modules/sekindoUMBidAdapter.js]: #1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like
```JavaScript
serverResponse = serverResponse.body;
```
just below this line, or however you'd prefer to grab the `body`, and updating corresponding tests if needed should get this back to working properly

File: modules/sekindoUMBidAdapter.js [hunk 3]
```diff
@@ -1,91 +1,113 @@
...
+  code: 'sekindoUM',
+  supportedMediaTypes: ['video'],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    if (bid.mediaType == 'video') {
```
Comment (matthewlane) [inline modules/sekindoUMBidAdapter.js]: As of #1483, video ad units may be defined with `mediaTypes: {video: {}}` and would be on this `bid` parameter, you'll probably want to check for that as well

File: modules/sekindoUMBidAdapter.js [hunk 4]
```diff
@@ -1,91 +1,113 @@
...
+      queryString = utils.tryAppendQueryString(queryString, 's', spaceId);
+      queryString = utils.tryAppendQueryString(queryString, 'subId', subId);
+      queryString = utils.tryAppendQueryString(queryString, 'pubUrl', pubUrl);
+      queryString = utils.tryAppendQueryString(queryString, 'hbTId', bidRequest.transactionId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbBidId', bidRequest.bidId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbver', '4');
+      queryString = utils.tryAppendQueryString(queryString, 'hbcb', '1');/// legasy
+      queryString = utils.tryAppendQueryString(queryString, 'dcpmflr', bidfloor);
+      queryString = utils.tryAppendQueryString(queryString, 'protocol', protocol);
+      if (bidRequest.mediaType === 'video') {
```
Comment (matthewlane) [inline modules/sekindoUMBidAdapter.js]: May want to check for `mediaTypes.video` as well, see other comment for related info

File: modules/sekindoUMBidAdapter.js (hunk 1 above)
Comment (nissSK) [reply modules/sekindoUMBidAdapter.js]: done

File: modules/sekindoUMBidAdapter.js (hunk 3 above)
Comment (nissSK) [reply modules/sekindoUMBidAdapter.js]: done

File: modules/sekindoUMBidAdapter.js (hunk 4 above)
Comment (nissSK) [reply modules/sekindoUMBidAdapter.js]: done

File: modules/sekindoUMBidAdapter.js (hunk 2 above)
Comment (nissSK) [reply modules/sekindoUMBidAdapter.js]: done

**Comment by jsnellbaker** [comment]
@nissSK I saw your updates to the feedback from @matthewlane, but I don't see that the changes were committed.  Are those changes still in the works on your end?  

Please confirm when you get the chance.  Thanks!

**Comment by nissSK** [comment]
@jsnellbaker Should be ok now (pushed changes to prebid1.0 instead of master)

**Comment by matthewlane** [comment]
@nissSK Changes look good but the 405 file changes are a bit concerning. Looks chmod related, if those can be resolved so there are just the 3 sekindo file changes we should be good for merge

**Comment by nissSK** [comment]
@matthewlane Sure, didnt notice that.

**Pull request merged**